src/redux/slices/patient/createPatient/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  firstName: '',
+  lastName: '',
+  email: '',
+  phoneNumber: '',
+  gender: '',
+  country: '',
+  city: '',
+  birthDate: '',
+  address: '',
+  timeZone: '',
+  overview: '',
+  isLoading: false,
+  token: null,
+  error: null,
+};
```

Comment:
can you make init state `{isLoading: false,}`? instead of describe each field as empty line

Reply:
fixed